server.js

```diff
@@ -243,13 +243,13 @@ app.get('/studentlist', function(req, res) {
     }
 
     if (typeof req.query.active === 'undefined') {
-        return res.status(500).send('Could not fetch student list');
+        return res.status(500).send(common.getError(2002));
     }
 
     /* only fetch student list once, then store it */
     users.getStudentsListWithStatus(req.query.active === 'true', function(err, studentlist) {
         if (err) {
```

Comment:
lets take the chance to log any non logged error like this one

Reply:
done for all